velox/vector/SelectivityVector.h

```diff
@@ -77,6 +78,16 @@ class SelectivityVector {
     updateBounds();
   }
 
+  // Sets the size and leaves content uninitialized and begin and end
+  // set to 0. The contents must be overwritten after calling this.
+  void resizeUnselected(int32_t size) {
```

Comment:
It is confusing to have method alongside `resize(int32_t size, bool value = true)` as it is not clear what are the differences between the two. Also, it seems like this method is  somewhat dangerous. For example, calling resizeUnselected(10) + setValid(5, true) + updateBounds() produces undefined results because bits from 0-10 except for 5 have undefined state.